sympy/solvers/ode/ode.py

```diff
@@ -1017,42 +1017,54 @@ class in it.  Note that a hint may do this anyway if
     # Any ODE that can be solved with a combination of algebra and
     # integrals e.g.:
     # d^3/dx^3(x y) = F(x)
+    solver_map = {
```

Comment:
This should probably be a module level constant. Perhaps it should be defined at the bottom of `single.py`.